src/lib/Icons/ChevronIcon.tsx

```diff
@@ -0,0 +1,35 @@
+import { color } from "@artsy/palette"
+import React from "react"
+import Svg, { G, Path } from "react-native-svg"
+
+const ChevronIcon = props => {
```

Comment:
More of a stylistic suggestion but a good pattern is to use object destructuring on `props` so `({expanded, initialDirection, width, height}) => `